README.md

```diff
@@ -175,26 +175,35 @@ spark.sql("delete from ${database}.${table} where xxx").show
+| `spark.tispark.tikv.key_cert_chain`                         | ``               | An X.509 certificate chain file for TiKV Client, e.g. `/home/tispark/config/client.crt`. If enable TLS, must be provided.                                                                                                                                                                                               |
+| `spark.tispark.tikv.key_file`                               | ``               | A PKCS#8 private key file for TiKV Client. e.g. `/home/tispark/client_pkcs8.key`. If enable TLS, must be provided.                                                                                                                                                                                                      |
+| `spark.sql.catalog.tidb_catalog.tikv.tls_enable`            | `false`          | Whether to enable TiSpark TLS for catalog.                                                                                                                                                                                                                                                                            |
+| `spark.sql.catalog.tidb_catalog.tikv.trust_cert_collection` | ``               | The same as `spark.tispark.tikv.trust_cert_collection`, e.g. `/home/tispark/config/root.crt`.                                                                                                                                                                                                                         |
+| `spark.sql.catalog.tidb_catalog.tikv.key_cert_chain`        | ``               | The same as `spark.tispark.tikv.key_cert_chain`, e.g. `/home/tispark/config/client.crt`.                                                                                                                                                                                                                               |
+| `spark.sql.catalog.tidb_catalog.tikv.key_file`              | ``               | The same as `spark.tispark.tikv.key_file`, e.g. `/home/tispark/client_pkcs8.key`.                                                                                                                                                                                                                                       |
+| `spark.tispark.enableJDBCSSL`                               | `false`          | Whether to enable TLS when use JDBC connector.                                                                                                                                                                                                                                                                        |
```

Comment:
I propose to add a sub-sector under Configuration. We need to tell when to use what properties and which properties should be combined to use.

Reply:
@zhangyangyu Thanks for your review. I have added some instructions and a configuration instance for open TLS.